addons/purchase/wizard/purchase_make_bill_advance.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, _, SUPERUSER_ID
+
+
+class PurchaseAdvancePaymentBill(models.TransientModel):
+    _name = 'purchase.advance.payment.bill'
+    _description = "Purchase Advance Payment Bill"
+    _inherit = ["account.advance.payment"]
+
+    amount_to_invoice = fields.Monetary(help="The amount to invoice = Purchase Order Total - Confirmed Down Payments.")
+    order_ids = fields.Many2many('purchase.order')
+
+    def view_draft_invoices(self):
```

Comment:
invoices or bills ?
There are some other examples in the files, could you please double check all utilisations of invoices vs bills (and use account_move if you need to include both, for instance in the abstract mixin)

Reply:
:up:

Reply:
At least the ones that are displayed to the user, for instance ` _('Draft Invoices'),`